tcptracer-bpf.c

```diff
@@ -30,6 +30,7 @@
 		bpf_trace_printk(____fmt, sizeof(____fmt), ##__VA_ARGS__); \
 	})
 
+#define TO_HEX(i) (i <= 9 ? '0' + i : 'A' - 10 + i)
```

Comment:
I couldn't find usages of this macro.

Reply:
That was debug code I added. Will remove.